bot/bot.py

```diff
@@ -27,6 +27,10 @@ def load_extensions(self) -> None:
     def run(self) -> None:
         """Run the bot with the token in constants.py/.env ."""
         logger.info("Starting bot")
+        if constants.TOKEN is None:
+            raise EnvironmentError(
+                "token value is None, make sure you have it set in .env"
```

Comment:
```suggestion
                "token value is None. Make sure you have it set in .env"
```
Minor grammar fix. Maybe there’s a better way to say “have it set.” That doesn’t *quite* work in my head. Something like “make sure you have configured the TOKEN field in .env” might be clearer.